scrapy/http/response/__init__.py

```diff
@@ -133,3 +133,23 @@ def follow(self, url, callback=None, method='GET', headers=None, body=None,
                        priority=priority,
                        dont_filter=dont_filter,
                        errback=errback)
+
+    def retry_request(self, reason=None, **kwargs):
```

Comment:
Here too, there's `Response.follow()` which returns a `Request`. Why not `Response.retry()`?